www/content/claviers/_index.md

```diff
@@ -171,6 +173,14 @@ sont plus faciles à prendre en main.
 Plus le clavier est compact, plus il est efficace et moins il est cher, au
 détriment du temps d’adaptation.
 
+### Quel <i lang="en">thumb cluster</i> ?
+
+Pour qu’un clavier soit utilisable facilement, il faut **au moins trois touches
+par pouce** qui soient bien accessibles ; quatre c’est mieux, cinq c’est large.
+
+Les claviers avec deux touches par pouce voire moins sont parfaits quand les
+[layer-taps][LT] et [homerow-mods][HRM] sont maitrisés.
```

Comment:
J’hésite à ouvrir la boîte de Pandore de la réforme orthographique de 1990 (<https://fr.wikipedia.org/wiki/Rectifications_orthographiques_du_fran%C3%A7ais_en_1990>). La pratique du site n’est pas tout à fait cohérente. Ce n’est pas très grave. Un correcteur bien réglé permettrait de voir assez vite les incohérences. À voir.

Reply:
J’écris sans correcteur, j’ai été formé à l’ancienne orthographe, mais je suis totalement en faveur de l’adoption de cette réforme (ognon, nénufar, téléférique). Si tu as le temps, la motivation et l’énergie de faire une passe sur tout le site ça m’intéresse — d’autant que tu risques d’apporter bien d’autres améliorations rédactionnelles dans l’élan. :-)

Reply:
C’est noté dans la mythique liste de tâches des vacances !